vdgsa_backend/rental_viols/models.py

```diff
@@ -1,3 +1,266 @@
+from __future__ import annotations
 from django.db import models
+from django.utils import timezone
 
-# Create your models here.
+
+class Bow(models.Model):
+    bow_num = models.AutoField(primary_key=True)
+    vdgsa_number = models.IntegerField(blank=True, null=True)
+    maker = models.CharField(max_length=50, blank=True, null=True)
+    size = models.CharField(max_length=9, blank=True, null=True)
+    state = models.CharField(max_length=10, blank=True, null=True)
+    value = models.DecimalField(max_digits=6, decimal_places=2, blank=True, null=True)
+    provenance = models.TextField(blank=True, null=True)
+    description = models.TextField(blank=True, null=True)
+    accession_date = models.DateField(blank=True, null=True)
+    notes = models.TextField(blank=True, null=True)
+    viol_num = models.ForeignKey(
+        'Viol', db_column='viol_num',
+        on_delete=models.PROTECT, blank=True, null=True
+    )
+    storer = models.IntegerField(blank=True, null=True)
+
+    program = models.ForeignKey(
```

Comment:
How often do we expect to add new "programs"? Suggest using a TextField with TextChoices (essentially an enum, see MembershipType in accounts/models.py)